src/Builtins/String.cpp

```diff
@@ -59,18 +59,44 @@ namespace Ark::internal::Builtins::String
 
     Value removeAtStr(std::vector<Value>& n, Ark::VM* vm)
     {
-        if (n.size () != 2)
+        if (n.size() != 2)
             throw std::runtime_error(STR_RM_ARITY);
         if (n[0].valueType() != ValueType::String)
             throw Ark::TypeError(STR_RM_TE0);
         if (n[1].valueType() != ValueType::Number)
             throw Ark::TypeError(STR_RM_TE1);
 
         long id = static_cast<long>(n[1].number());
-        if (id < 0 || id >= n[0].stringRef().size())
+        if (id < 0 || static_cast<std::size_t>(id) >= n[0].stringRef().size())
             throw std::runtime_error(STR_RM_OOR);
 
         n[0].stringRef().erase(id, id + 1);
         return n[0];
     }
+
+    Value ord(std::vector<Value>& n, Ark::VM* vm)
+    {
+        if (n.size() != 1)
+            throw std::runtime_error("str:ord needs at least 1 argument: string");
+        if (n[0].valueType() != ValueType::String)
+            throw Ark::TypeError("str:ord: string must be a String");
```

Comment:
can you put the text of the exceptions inside the builtins.inl as we do for the other? following the same naming convention, so that all error messages are in the same place

Reply:
of course